src/normalizeCqlExecution.ts

```diff
@@ -2,15 +2,34 @@ import { Uri, window } from 'vscode';
 import { buildParameters } from './buildParameters';
 import { executeCQL } from './executeCql';
 
-export async function normalizeCqlExecution(uri: Uri, mode: 'expression' | 'file') {
-  // Needs a distinction between CQL file and single line
+export async function normalizeCqlExecution(uri: Uri, type: string) {
   const isCqlFile = window.activeTextEditor!.document.fileName.endsWith('.cql');
-
-  if (isCqlFile && mode === 'file') {
-    // should normalize data
-    let operationArgs = buildParameters(uri);
-    executeCQL(operationArgs);
+  let operationArgs;
+  if (isCqlFile && type === "file") {
+    operationArgs = buildParameters(uri, undefined);
+  } else if (isCqlFile && type === "expression") {
+    let cursorPosition = window.activeTextEditor!.selection.active;
+    const definitionStatementRegex = new RegExp('(?<=\\bdefine\\s*"\\s*)([^"]+)(?="\\s*:)');
```

Comment:
Please leave a comment here stating that this is intended to be temporary, and that we hope to later use the language server functionality for parsing.

Reply:
added the comment